os/board/rtl8721csm/src/component/os/tizenrt/rtk_netmgr.c

```diff
@@ -318,6 +319,22 @@ trwifi_result_e wifi_netmgr_utils_scan_ap(struct netdev *dev, trwifi_ap_config_s
 	return wuret;
 }
 
+trwifi_result_e wifi_netmgr_utils_single_channel_scan(struct netdev *dev, trwifi_scan_config_s *scan_config)
+{
+	trwifi_result_e wuret = TRWIFI_INVALID_ARGS;
+	if (!scan_config) {
+		return wuret;
+	}
+
+	wuret = TRWIFI_FAIL;

```

Comment:
I think we don't need to set the `wuret` for every case, we can just return directly. Then we can save much code.

Reply:
Done.